pjsip-apps/src/pjsua/pjsua_app_config.c

```diff
@@ -180,6 +180,11 @@ static void usage(void)
     puts  ("  --auto-play-avi     Automatically play the AVI media to call");
 #endif
 
+    puts  ("");
+    puts  ("Text Options:");
+    puts  ("  --text              Enable real-time text stream");
+    puts  ("  --text-red=N        Text stream redundancy level (default=0)");
```

Comment:
the default seems to be 2?

Reply:
For PJSUA-LIB, the default is 2.
But for pjsua app, if not specified, the default is 0 (no redundancy will be offered).

This may be kind of similar to video and text settings itself. In PJSUA-LIB, the default of vid_cnt and txt_cnt is 1, but in pjsua app it's 0.